[2021-12-21 22:37:35,737] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline.sketch scheduled__2021-12-19T00:01:00+00:00 [queued]>
[2021-12-21 22:37:35,741] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline.sketch scheduled__2021-12-19T00:01:00+00:00 [queued]>
[2021-12-21 22:37:35,742] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-21 22:37:35,742] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-21 22:37:35,742] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-21 22:37:35,751] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): sketch> on 2021-12-19 00:01:00+00:00
[2021-12-21 22:37:35,752] {standard_task_runner.py:52} INFO - Started process 98803 to run task
[2021-12-21 22:37:35,754] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline', 'sketch', 'scheduled__2021-12-19T00:01:00+00:00', '--job-id', '99', '--raw', '--subdir', '/home/myamya/project/dockers/airflow/dags/hello_world.py', '--cfg-path', '/tmp/tmp4vhga22a', '--error-file', '/tmp/tmpwi1s2bcx']
[2021-12-21 22:37:35,754] {standard_task_runner.py:77} INFO - Job 99: Subtask sketch
[2021-12-21 22:37:35,790] {logging_mixin.py:109} INFO - Running <TaskInstance: data_pipeline.sketch scheduled__2021-12-19T00:01:00+00:00 [running]> on host myamya
[2021-12-21 22:37:35,818] {taskinstance.py:1412} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yangjae
AIRFLOW_CTX_DAG_ID=data_pipeline
AIRFLOW_CTX_TASK_ID=sketch
AIRFLOW_CTX_EXECUTION_DATE=2021-12-19T00:01:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-19T00:01:00+00:00
[2021-12-21 22:37:35,818] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-21 22:37:35,819] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'docker run -it -v /home/myamya/project/image_files/:/home/myamya/project/image_files/ sketch:0.4 /bin/bash /home/myamya/project/init.sh ']
[2021-12-21 22:37:35,822] {subprocess.py:85} INFO - Output:
[2021-12-21 22:38:04,722] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-21 22:38:04,749] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=sketch, execution_date=20211219T000100, start_date=20211221T133735, end_date=20211221T133804
[2021-12-21 22:38:04,781] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-21 22:38:04,797] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
